frontend/sphinx/conf.py

```diff
@@ -322,6 +329,7 @@ def get_file_from_conf_ini(path_to_file):
     'preamble': r'''
 \usepackage{pmboxdraw} \usepackage{unicode-math}
 \usepackage{pdfpages}
+\usepackage{pdflscape}
```

Comment:
What's the reason for introducing this package?

If I understood it correctly, it's supposed to support pages printed/displayed in landscape format. However, all other courses use the portrait format, and it works fine, so I don't see a reason to have a different format in this specific case.